tests/logic/test_logic.py

```diff
@@ -175,3 +175,34 @@ def tearDown(self):
             db.session.remove()
             db.drop_all()
             db.engine.dispose()
+
+class HelpersTest(unittest.TestCase):
+
+
+    def setUp(self):
+        self.descriptor = json.loads(open('fixtures/datapackage.json').read())
+
+
+    def test_validate_for_jinja_returns_descriptor_if_no_licenses(self):
+        descriptor = validate_for_template(self.descriptor)
+        self.assertEqual(self.descriptor, descriptor)
+
+    def test_validate_for_jinja_returns_descriptor_if_licenses_is_list(self):
+        self.descriptor['licenses'] = []
+        descriptor = validate_for_template(self.descriptor)
+        self.assertEqual(self.descriptor, descriptor)
+
+    def test_validate_for_jinja_modifies_descriptor_if_licenses_is_dict(self):
+        self.descriptor['licenses'] = {'url': 'test/url', 'type': 'Test'}
+        descriptor = validate_for_template(self.descriptor)
+        self.assertEqual(descriptor['license'], 'Test')
```

Comment:
so `licenses` (plural) then it is a list, and if `license` singular then it is a string?